internal/provider/models/processors/aggregate_v2.go

```diff
@@ -38,21 +40,32 @@ var AggregateV2ProcessorResourceSchema = schema.Schema{
 		},
 		"interval": schema.Int64Attribute{
 			Optional:    true,
-			Description: "When method is set to tumbling, this is the interval over which metrics are aggregated in millseconds",
+			Computed:    true,
+			Description: "When method is set to tumbling, this is the interval over which metrics are aggregated in seconds",
 		},
 		"strategy": schema.StringAttribute{
 			Required:    true,
 			Description: "When method is set to sliding, this is the strategy in which to perform the aggregation",
 		},
 		"window_duration": schema.Int64Attribute{
 			Optional:    true,
+			Computed:    true,
 			Description: "When method is set to sliding, this is the interval over which metrics are aggregated in seconds",
 		},
 		"conditional": schema.SingleNestedAttribute{
 			Optional:    true,
 			Description: "When method is set to sliding: " + ParentConditionalAttribute.Description,
 			Attributes:  ParentConditionalAttribute.Attributes,
 		},
+		"group_by": schema.ListAttribute{
+			ElementType: basetypes.StringType{},
+			Optional:    true,
+			Computed:    true,
+			Description: "Group events based on matching data from each of these field paths. Supports nesting via dot-notation.",
+			Default: listdefault.StaticValue(
```

Comment:
Given that this is a computed field, should we leave setting the default value to the pipeline service API?